### ISSUE TITLE
Differential learning rates

### ISSUE THREAD

**Issue by tom-adsfund** [deeplearning4j/deeplearning4j#4606]
In the recent Fast.ai course, there is an example of using ResNet via transfer learning and with differential learning rates. The results were impressive as could be expected (different parts of the network -- those towards the output -- are more likely to require change).

I'd really like to do the same in DL4J. I'm not sure if this is possible, but from a glance at the Javadocs it seems not.

Being able to do this with Resnet50 would allow many powerful applications, especially from Java.

**Comment by AlexDBlack** [comment]
Transfer learning allows you to set a FineTuneConfiguration to specify the learning rate (and everything else) for the existing/transferred net, and then you can override/specify it for the new layers.

There's also this new (post 0.9.1) functionality here:
https://github.com/deeplearning4j/deeplearning4j/blob/8e132b6e7c0f530f8bcfae82dc961e15f905e8fe/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java#L3358-L3402

So unless we're missing something, I'll close this.

**Comment by tom-adsfund** [comment]
Great, thanks for the explanation.
